config/locales/en/pages.yml

```diff
@@ -7,3 +7,7 @@ en:
           <p>
             You will be able to use the service again once we resolve the problem we’ve found.
           </p>
+          <p>
+            Use <a href="https://portal.legalservices.gov.uk/"><abbr title='Client and Cost Management System'>CCMS</abbr> (Client and Cost Management System)</a>
```

Comment:
will need to be changed to use `Rails.configuration.x.laa_landing_page_target_url` for the new "portal" replacement service